components/Comment/lib/api.ts

```diff
@@ -1,16 +1,18 @@
-import {
-  ID,
-  RHUser,
-  RhDocumentType,
-  parseUser,
-} from "~/config/types/root_types";
+import { ID, RhDocumentType } from "~/config/types/root_types";
```

Comment:
Please try to keep the imports and arguments in alphabethical order. If you use VS code, you can highlight the lines and press Fn+F9, it alphabetizes for you